### DISCUSSION TITLE
October 23, 2021

### DISCUSSION THREAD

**Discussion by manuelalferez** [pmascaraque/iycdangbo#4] [Meetings]
# Meetings notes:

**Comment by manuelalferez** [comment]
- https://pudding.cool/2020/08/campaign-colors
- https://stripe.com/docs
- https://vercel.com/
- https://stripe.com/docs
- https://hexo.io/docs/
- https://docs.manim.community/en/stable/index.html

**Comment by jesi-rgb** [comment]
Ejemplo de interactive table of contents para cuando hay mucho texto

- https://docs.manim.community/en/stable/installation/troubleshooting.html#identifying-files-written-for-a-different-version-of-manim

**Comment by pmascaraque** [comment]
https://www.figma.com/file/waOmZIc3ZIgSE2HluNNMLL/IyCDangbo?node-id=0%3A1
